$NetBSD: patch-Makefile,v 1.1 2019/07/09 15:46:42 riastradh Exp $

Define CC_FOR_BUILD so we can use it even when not cross-compiling.

--- Makefile.orig	2019-07-08 17:03:10.000000000 +0000
+++ Makefile
@@ -95,6 +95,7 @@ CFLAGS1 = $(CFLAGS0) #-posix -Xp
 LDFLAGS1= $(LDFLAGS0) $(LIBS) #-lcposix
 
 ####CC	= cc # gcc
+CC_FOR_BUILD = $(CC)
 # object file extension
 O	= o
 RM	= /bin/rm -f
